tests/test_authz_decorator.py

```diff
@@ -49,15 +49,21 @@ def test_root_permissions_success(
         self, sample_resource_with_authorization: type[Resource], full_access_auth_header: str
     ) -> None:
         res_instance = sample_resource_with_authorization(
-            APIGatewayEvent("/", "GET", headers={"authorization": full_access_auth_header})
+            APIGatewayEvent(
+                resource_path="/", method="GET", headers={"authorization": full_access_auth_header}
+            )
```

Comment:
Yes, the order of arguments has been changed but you should follow it instead of adding keywords:
```suggestion
            APIGatewayEvent("GET", "/", headers={"authorization": full_access_auth_header})
```